### ISSUE TITLE
hw01 ready for grading

### ISSUE THREAD

**Issue by gbraich** [gbraich/STAT545-hw01-Braich-Gurneet#1]
@vincenzocoia @gvdr @ksedivyhaley @joeybernhardt @mynamedaike @pgonzaleze @derekcho

**Comment by hannahdxz** [comment]
Hi Gurneet! Nice to meet you! Thank you for sharing your readme, and here is some comments:

Your readme demonstrates experimentation with more than four aspects of the Markdown syntax, such as section headers, links, bold, italic, etc. It very well formatted!  You reflected on your experience with R markdown and how you got the changes into README.md on GitHub. I like that you shared your experience by putting down the link of the tutorial you find helpful. A little suggestion maybe to add one or two pictures to make the readme file looks more colorful! 

Moreover I saw that you explored the Gapminder dataset and rendered markdown file.  A suggestion might be to add some notes or explanation for each code chunk. 

Overall, great work and I hope you enjoy this class!

**Comment by aiod01** [comment]
Hello Gurneet, I am Hyeongcheol Park. Thank you for your inviting me on your GitHub. 

 the readme.md file was fascinating. Especially, I learned some code from yours, which is the thicker horizontal line( *** ), and quote (>). I like that part  I couldn't find on my information area. I would love to use those in my next md file!

  Your readme file has many aspects, such the line( by using ***), quote(>)  dot(*), quotation mark(""), header(###), link(![] )and so on. it seems you are very good at using the * !. And what I like on the text is the last part, the process, which leads me to remind what I learned from the class again. 

 A tiny suggestion, if I have to give, would be to add pictures or gif, or any other emoticons. Or you may try some change on your bullet point, to give numbering or try some different bullet point such as + or -. 

 Regarding your Gapminder exploring, it was nice of trying brand-new plot, ggplot! Coloring((color = continent)) and `geom_smooth()` using method = 'gam' made the file alive! I think it is awesome to try new thing as soon as possible to get the skill to be mine. 

 If I should give you some suggestion, it might be better to give more comments with ##, on your file to explain what the fancy graphs mean and how did you drive the coding to your document.

 To sum up, it was very nice work. I hope you keep enjoy learning!

**Comment by derekcho** [comment]
Hi @gbraich! Here are some comments about your hw01:

The README file looks excellent. Very neat and tidy with multiple markdown syntaxes used. Good description of your workflow as well!
For your hw01_gapminder.md file, it would be nice to have some text in between explaining what the code is doing at each step since it is hard for the reader to read lines and lines of code without much context. It would have been nice to see some extensions as well instead of exactly what was done in class. For example, you could try plotting the relationship between other variables. Good job!

Note: your mark (check minus, check, check plus) will be distributed later.
